app/models/ems_folder.rb

```diff
@@ -177,7 +162,7 @@ def folder_path_objs(*args)
     options = args.extract_options!
     folders = path(:of_type => "EmsFolder")
     folders = folders[1..-1] if options[:exclude_root_folder]
-    folders = folders.reject { |f| NON_DISPLAY_FOLDERS.include?(f.name) } if options[:exclude_non_display_folders]
+    folders = folders.reject(&:hidden?) if options[:exclude_non_display_folders]
```

Comment:
I don't know, that's a good question. I would guess we'd need to do a data migration but I don't know for sure. I do know a migration spec could tell you... And be the baseline test if you need to manually update the rows.